server/src/middlewares/webAppProxyMiddleware.ts

```diff
@@ -41,6 +41,13 @@ export async function metasResponseInterceptor(
   req: IncomingMessage,
   res: ServerResponse
 ) {
+  // When a js chunk is fetched, it is expected to have a content-type: application/javascript
+  // if content-type: text/html, it means the index.html fallback as been returned, in such case, we do not want any cache
+  if (proxyRes.headers['content-type'] === 'text/html' && req.url?.endsWith('.js')) {
+    // If this is still cached and thus not working, test with no-store instead of no-cache
+    res.setHeader('cache-control', 'public,no-cache')
```

Comment:
détails : il me semblait que les headers HTTP devait etre avec des majuscules

```suggestion
    res.setHeader('Cache-Control', 'public,no-cache')
```